src/foundry/client/config.d.mts

```diff
@@ -3731,6 +3426,440 @@ declare global {
         ago: typeof foundry.data.CalendarData.formatAgo;
       }
     }
+
+    namespace Wall {
+      /** @internal */
+      type _DoorSound = InexactPartial<{
+        /**
+         * One or more sound paths for when the door is closed
+         * @remarks If an array is provided, a random entry is chosen
+         */
+        close: string | string[];
+
+        /**
+         * One or more sound paths for when the door becomes locked
+         * @remarks If an array is provided, a random entry is chosen
+         */
+        lock: string | string[];
+
+        /**
+         * One or more sound paths for when opening the door
+         * @remarks If an array is provided, a random entry is chosen
+         */
+        open: string | string[];
+
+        /**
+         * One or more sound paths for when attempting to open a locked door
+         * @remarks If an array is provided, a random entry is chosen
+         */
+        test: string | string[];
+
+        /**
+         * One or more sound paths for when the door becomes unlocked
+         * @remarks If an array is provided, a random entry is chosen
+         */
+        unlock: string | string[];
+      }>;
+
+      interface DoorSound extends _DoorSound {
+        /** A localization string label */
+        label: string;
+      }
+
+      interface DoorSounds {
+        [sound: string]: DoorSound;
```

Comment:
I think we need to brand all index signatures based upon `CONFIG.Dice.RollModes`. Otherwise we're stuck in a situation where either we accept any string or doing `Object.keys(doorSounds)` won't let you use it as a valid sound.

Watch out I did update the template there slightly recently.